BEES.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# encoding: utf-8
```

Comment:
This commit (`Four new file on work : main, excutable, logger and common`) adds the Common.py and Logger.py files which are then deleted (or renamed) in this PR by later commits. It's better to commit it correctly to begin with.
(Also, this commit has `#TODO:`, we prefer not to merge such comments, but keep a todo list seperately)